src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -113,19 +113,37 @@ protected function get_properties() {
 					),
 				),
 			),
-			'price'      => array(
+			'price'         => array(
 				'description' => __( 'Current product price.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => array( 'view', 'edit' ),
 				'readonly'    => true,
 			),
-			'line_price' => array(
-				'description' => __( 'Current line price.', 'woo-gutenberg-products-block' ),
+			'line_price'    => array(
+				'description' => __( 'Line price subtotal (excluding any discounts).', 'woo-gutenberg-products-block' ),
```

Comment:
We may need to clear up our language here between `_price` and `_subtotal`. Core uses `_subtotal` and `_total` terminology in the cart. In this case, `line_price` is the subtotal, because it's before discounts.

Perhaps `line_price` should become `line_subtotal`, and `line_subtotal` (in this PR) should be `line_total`?

Reply:
I agree, I think it's a good idea to review field names for consistency. Will make some changes  as you've suggested :)

Reply:
I also renamed `price` to `product_price` (alternative: `item_price`).

<img width="781" alt="Screen Shot 2019-12-09 at 11 33 42 AM" src="https://user-images.githubusercontent.com/4167300/70397517-c4003600-1a77-11ea-991b-5ad2d8bca0f3.png">